.github/workflows/activation.yml

```diff
@@ -0,0 +1,19 @@
+name: Acquire activation file
+on: [push]
+jobs:
+  activation:
+    name: Request manual activation file 🔑
+    runs-on: ubuntu-latest
+    steps:
+    # Request manual activation file
+    - name: Request manual activation file
+      id: getManualLicenseFile
+      uses: webbertakken/unity-request-manual-activation-file@v1.1
+      with:
+        unityVersion: 2019.4.12f1

```

Comment:
You targeted version 2019.4.12f1, but you also pushed "ProjectVersion.txt" with 2019.4.13f1.